src/commands/help.ts

```diff
@@ -60,6 +60,18 @@ module.exports = {
       ["npx ignite-cli remove-demo-markup", "npx ignite-cli remove-demo-markup --dry-run"],
     )
     p()
+    command(
+      "remove-mst (rm-mst)",
+      "Removes MobX-State-Tree code from the project (add --dry-run to list changes but not execute)",
+      ["npx ignite-cli remove-mst", "npx ignite-cli remove-mst --dry-run"],
+    )
+    p()
+    command(
+      "remove-mst-markup (rmstm)",
```

Comment:
From what I can tell, removal of MST, and removal of MST markup can only be done when generating a new project, since either way the markup won't exist after generation, right?

If that's the case, should we consider these commands "internal" and not document them here, since they can't be used by end users? Or at least add that information in to the docs for context.

Reply:
I can't think of any reason, no, @frankcalise ?

Reply:
I think the reasoning was if you had an existing project where the markup wasn't removed (for the demo markup that is). We didn't always remove the markup out of the box.

That was probably around v8 though and fixed in or around some patch of 9 - we could probably look to remove it as never leaving mark up is what's current.

If we want to address the demo one as a separate PR that's fine